src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -20,8 +20,15 @@ module swarm.neo.client.mixins.RequestCore;
 
 public enum RequestType
 {
+    // A request that operates on one node at a time. The node currently being
+    // operated on may change over the request's lifetime.
     SingleNode,
+
+    // A request that operates on one node at a time. If the request fails on
+    // one node, it is repeated on the next in a round-robin sequence.
```

Comment:
Does the request need to fail to be repeated on the next in a round-robin sequence? Is there are request that always tries on the next node in a round-robin sequence (like `Produce`)?

Reply:
I think currently not, but I can make the comment more generic.